app/scripts/satisfaction.js

```diff
@@ -1,11 +1,20 @@
-window.onload = function () {
+jQuery(document).ready(function($) {
   var satisfaction = getQueryVariable('satisfaction');
   var origin = getQueryVariable('origin');
   document.getElementById('result').value = satisfaction;
   document.getElementById('origin').value = origin;
   document.getElementById('ip').value = myip;
-  document.getElementById('satisfaction').submit();
-}
+});
+$( window ).load(function() {
```

Comment:
Why are you referencing jQuery with a $ here now?
Why are you wrapping window in jQuery?